sdk/go/fs.go

```diff
@@ -23,3 +23,6 @@ var GoSum []byte
 
 //go:embed engineconn/*.go querybuilder/marshal.go querybuilder/querybuilder.go go.mod go.sum client.go dagger.gen.go telemetry/*.go
 var GoSDK embed.FS
+
+//go:embed dagger.gen.go
+var GoDagGen []byte
```

Comment:
Struggling to understand why we need this - can we not just generate this in the binary? 

I guess it is a speed-up (?), but it does feel like it just *expands* the number of possible paths through the `codegen` binaryh.

Also just feels a little strange that the binary that generates this file can embed itself. Maybe we could just put this file *into* the go runtime container?

Reply:
Yes, the idea is typedefs should only require the core types + module source code to be generated. They do not need dependencies for instance. By using the already generated code, we avoid to generate the sdk just for the typedefs.